common/src/main/java/net/william278/husksync/data/DataServerID.java

```diff
@@ -0,0 +1,37 @@
+package net.william278.husksync.data;
+
+import com.google.gson.annotations.SerializedName;
+import org.jetbrains.annotations.NotNull;
+
+public class DataServerID {
+
+    @SerializedName("server_id")
```

Comment:
Are we serializing this anywhere? Not sure this is needed unless we are, and I think we're just storing the server as a string.